apps/web/components/new-experiment.tsx

```diff
@@ -46,9 +46,19 @@ export function NewExperimentForm() {
   }
 
   async function onSubmit(data: CreateExperimentBody) {
```

Comment:
This should not be needed since the query client should do this 
```

// Create a client-side QueryClient instance
const queryClient = new QueryClient({
  defaultOptions: {
    mutations: {
      onError: (error) => {
        toast({
          description: error.message,
          title: error.name,
          variant: "destructive",
        });
      },
    },
  },
});
```
 if that is not working that should be fixed